scripts/make-spritesheet.js

```diff
@@ -0,0 +1,63 @@
+/* global __dirname process */
+
+require("dotenv").config({ path: "src/.env" });
+
+const fs = require("fs-extra");
+const Spritesmith = require("spritesmith");
+const glob = require("glob");
+const path = require("path");
+
+const distMarkersPath = path.resolve(
+  __dirname,
+  "../www/static/css/images/markers",
+);
+
+const markers = glob.sync(
+  path.resolve(distMarkersPath, "*.{png,jpg,jpeg,gif}"),
+);
+
+Spritesmith.run({ src: markers }, (err, result) => {
+  if (err) {
+    console.log("Spritesmith error:", err);
+
+    process.exitCode = 1;
+    process.exit();
+  }
+
+  fs.writeFileSync(
+    path.resolve(distMarkersPath, "spritesheet.png"),
+    result.image,
+  );
+  fs.writeFileSync(
+    // For high-resolution devices; Mapbox expects this file.
+    path.resolve(distMarkersPath, "spritesheet@2x.png"),
+    result.image,
+  );
+
+  // Postprocess the coordinates object.
+  Object.keys(result.coordinates).forEach(spriteIdentifier => {
```

Comment:
nit: can we use a `map` here? It's much easier to understand code, and safer, when we aren't mutating state.